demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -808,35 +800,87 @@ static void otaAppCallback( OtaJobEvent_t event,
             LogDebug( ( "Received invalid callback event from OTA Agent." ) );
     }
 }
+
+jobMessageType_t getJobMessageType( const char * pTopicName,
```

Comment:
These functions can be preferably moved to helper functions or within mqtt implementation of lib directory.

Reply:
This will be covered in another PR.